### ISSUE TITLE
CWL: Argue to CWL team to change spec to be container runtime neutral

### ISSUE THREAD

**Issue by rstyd** [lanl/BEE#270]
Currently, the CWL spec's container support is entirely focused on docker. I think this was reasonable at the time considering big hitters like podman/buildah were in relative infancy ~3 years ago when CWL was first developed. The landscape has changed considerably since then though. It'd be very helpful if we could convince them to generalize their container interface to use any OCI compliant runtime. I don't think it would require much work on their part other than changing the names of fields like `dockerLoad` and `dockerImport` to `containerLoad` and `containerImport` since by definition OCI runtimes work _like_ docker. What do you all think?

**Comment by pagrubel** [comment]
@Boogie3D This issue is old but relevant to collaborating on the standard, getting BEE as an implementation is more important and adding some of our extensions.

**Comment by mr-c** [comment]
Yes, it would be great to merge more of your extensions into CWL!

FYI, CWL has always been neutral on the software container runtime; just that the format for the container is the Docker format. Various CWL implementations use podman, apptainer, singularity, udocker, etc.. and pass all the CWL conformance tests.

**Comment by pagrubel** [comment]
@mr-c Thank you for your input we do have this on our to do list.
